### DISCUSSION TITLE
Support for Flux.1

### DISCUSSION THREAD

**Discussion by CorentinWicht** [LostRuins/koboldcpp#1085] [Ideas]
Dear @LostRuins,

[/leejet/stable-diffusion.cpp](https://github.com/leejet/stable-diffusion.cpp/) has [recently been updated](https://github.com/leejet/stable-diffusion.cpp/pull/356) to support the new image generation model Flux.1 (see [this thread](https://github.com/leejet/stable-diffusion.cpp/issues/323)).

Would it be possible to update `stable-diffusion.cpp` in the upcoming koboldcpp release so that we can try it out? 

Any recommendation on which model should be favoured (Dev/Schnell, quantization, etc.) ?

Best,

C.

**Comment by LostRuins** [comment]
I'll consider doing it eventually, but it might be quite some time - there are many difference between the current live branch of sd.cpp and the one used in kcpp and reconciling them will take a lot of time - especially since I also can't test flux locally due to lack of VRAM. But it would be nice to have.
